src/plugins/tbin/tbin/Map.hpp

```diff
@@ -42,10 +42,10 @@ namespace tbin
         public:
             bool loadFromFile( const std::string& path );
             bool loadFromStream( std::istream& in );
-            
```

Comment:
Please don't change any files in the `tbin/tbin` folder, they're 3rd party code.